packages/taboule/src/state/queries.ts

```diff
@@ -138,10 +144,25 @@ export const GetTabouleQueries = ({
   >(
     (input) =>
       pipe(
-        YTAPI.v1.Public.GetPersonalStatsByPublicKey(input),
+        YTAPI.v2.Metadata.ListMetadata({
+          Query: {
+            ...input.Query,
+            amount: '20' as any,
+            skip: '0' as any,
+            experimentId: undefined,
+            researchTag: undefined,
+            format: 'json',
+            nature: 'home',
+            publicKey: 'H7AsuUszehN4qKTj2GYYwNNzkJVqUQBRo2wgKevzeUwx',
```

Comment:
I would use the `publicKey` coming from the `input.Query` and remove the hardcoded value